app/models/task.py

```diff
@@ -2,4 +2,21 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default=None)
```

Comment:
Fun fact: the default value for nullable is `True` so we don't need to explicitly add it 😊